src/GoTrueAdminApi.ts

```diff
@@ -227,4 +235,8 @@ export default class GoTrueAdminApi {
       throw error
     }
   }
+
+  async _deleteFactor() {
```

Comment:
Mark it as `private` to be hidden from TypeScript users? (Note there's no such thing as "private" in JavaScript, it's only enforced by TypeScript, thus the `_` prefix.)